Create LimelightMath.java

EliTheCompSciGuy opened the pull request:
I didn't know the exact number for some variables or the exact format of the Limelight code, but I got the math for most (if not all) of the calculations to determine distances and their rate of change. I parsed whatever I didn't know or if the value would be imported from another class.

EliTheCompSciGuy committed:
Create LimelightMath

I didn't know the exact number for some variables or the exact format of the Limelight code, but I got the math for most (if not all) of the calculations to determine distances and their rate of change. I parsed whatever I didn't know or if the value would be imported from another class.

DanWaxman reviewed (changes requested):
The math itself looks good. However, a few things to change before merging:
- There's a lack of documentation for parameters and returns. Documentation is key to the usage of a lot of these methods.
- This class isn't in the package with the rest of the robot code.
- It utilizes an entrypoint and GUI components, neither of which are available on the robot (okay entrypoint maybe, but still not good practice). 

After these changes are made, it looks good and will be good to push. Good job!

EliTheCompSciGuy committed:
Update LimelightMath

I removed all of the parsing at the beginning of the code to prevent dialog boxes from popping up. I didn't put in any computations/values for these variables since x and v would be imported from another source and y hasn't been determined yet. I also put in documentation stating what each variable means and what each method is calculating .
